inventory.yml.sample

```diff
@@ -291,11 +291,32 @@ all:
 
               # # Uncomment to set an alternate installation disk device for the node
               # installation_disk_path: /dev/sdb
-
+              
+              # # Uncomment to set additional partitions if required on nodes. Please provide the correct label and size(MiB).
+              # # The device can either be the drive in which RHCOS image needs to be installed or it can be any additional drive that requires new partitions.
+              # # Add all additional paritions required under extra_partitions (examples for label:size provided below).
+              # # Additional partitions can only be created for OCP 4.8+ deployments
+              # disks:
+              # - device: "{{ installation_disk_path }}" #See above section
+              #   extra_partitions:
+              #     partition_1: 1024
+              #     partition_2: 102
+              
```

Comment:
This is an advanced feature and doesn't need to be in the sample inventory. The doc on how to use it should cover when, how, and where to use it.

Reply:
Change have been made. Removed from Sample inventory